packages/components/src/internal/components/security/APIWrapper.ts

```diff
@@ -30,6 +38,7 @@ export interface RemoveGroupMembersResponse {
 export interface SecurityAPIWrapper {
     addGroupMembers: (groupId: number, principalIds: number[], projectPath: string) => Promise<AddGroupMembersResponse>;
     createGroup: (groupName: string, projectPath: string) => Promise<Security.CreateGroupResponse>;
+    deleteContainer: (comment: string) => Promise<Record<string, unknown>>;
```

Comment:
Rather than specifying a new interface consider switching this to `deleteContainer: (options: Security.DeleteContainerOptions) => Promise<Record<string, unknown>>;` so it doesn't have to be updated in the future if additional options are made available.